django_ontruck/monads/error.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+from .result import Result
+
+from typing import Any, Callable, TypeVar, Optional, final
+
+
+T = TypeVar('T')
+U = TypeVar('U')
+
+E = TypeVar('E')
+F = TypeVar('F')
+
+
+@final
```

Comment:
Why the `final` here out of curiosity?

Reply:
Means that you can't override the class Ok/Error to do something else out of control. You have to take it as it is.